ci/build/Dockerfile

```diff
@@ -94,8 +94,14 @@ COPY ./ci/build/files /
 
 RUN curl -sS https://getcomposer.org/installer | php \
     && mv composer.phar /usr/local/bin/composer \
-    && chmod +x /usr/local/bin/composer \
-    && composer install --no-dev --optimize-autoloader --working-dir=/hypernode
+    && chmod +x /usr/local/bin/composer
+
+# Remove dev dependencies from composer.json
```

Comment:
We have the wierd case that only our dev dependencies (psalm) have the hard limit for PHP 8.1. There's no way to remove this check except entirely disabling the platform requirements. That's why we remove dev-require in the production docker build